### PR TITLE
doc: nRF53 UG: IEEE 802.15.4 design in multi-core SoC

### PR DESCRIPTION
**Pull request by AdamZelikNS**
- nRF53 UG update with a drawing showing the architecture of the 802.15.4
solution for multicore SoC.

### CODE REVIEW

**Review by hubertmis — APPROVED** [review]

**Review by jciupis — APPROVED** [review]
There's a typo in one of the new filenames. Please rename ` doc/nrf/images/ieee802154_nrf53_multiprot_desig.svg` to ` doc/nrf/images/ieee802154_nrf53_multiprot_design.svg`

File: doc/nrf/ug_nrf5340.rst [hunk 1]
```diff
@@ -153,15 +154,35 @@ For example:
 
   This configuration guarantees that the network sample can handle the Bluetooth LE DLE update procedure, which is used in the :ref:`ble_throughput` sample.
 
-IEEE 802.15.4 (Thread and Zigbee)
----------------------------------
+Single protocol IEEE 802.15.4 (Thread and Zigbee)
```
Comment (hubertmis) [inline doc/nrf/ug_nrf5340.rst:157]: I'm not sure if `Single protocol` part is necessary here. I think if we don't use `multiprotocol` word in this context it is clear that Thread/Zigbee is single protocol

**Review by hubertmis — APPROVED** [review]
Looks good!

**Comment by FrancescoSer** [comment]
@carlescufi Please note that this PR was tagged as doc-required but was merged without a tech writer review

**Pull request merged**